xmtp_mls/src/groups/subscriptions.rs

```diff
@@ -28,8 +28,14 @@ impl MlsGroup {
 
             // Attempt processing immediately, but fail if the message is not an Application Message
             // Returning an error should roll back the DB tx
-            self.process_message(&mut openmls_group, provider, &msgv1, false)
-                .map_err(GroupError::ReceiveError)
+            await_helper(self.process_message(
+                client.as_ref(),
+                &mut openmls_group,
+                provider,
+                &msgv1,
+                false,
+            ))
+            .map_err(GroupError::ReceiveError)
```

Comment:
I think the `await_helper` is what is crashing for me on android for all of streaming.